kafka-connect-s3/docs/configuration_options.rst

```diff
@@ -39,6 +39,20 @@ Connector
   * Default: 1000
   * Importance: low
 
+``enhanced.avro.schema.support``
```

Comment:
Why are we adding those? Where they missing or they are just carried over?

Reply:
In this case, they need to go on a separate PR that targets the appropriate `-post` branch

Reply:
They were missing  and autogenerated -- However the testing against them is weak so I'm not 100% sure they are implemented correctly. Do you have any additional insight? Looks like you added it last october